core/cap-0035.md

```diff
@@ -254,22 +255,42 @@ index 7f08d757..e0585994 100644
 +{
 +    ClaimableBalanceID balanceID;
 +};
++
++/* SetTrustlineFlagsOp
++
++   Updates the flags of an existing trust line.
++   This is called by the issuer of the related asset.
++
++   Threshold: low
++
++   Result: SetTrustlineFlagsResult
++*/
++struct SetTrustlineFlagsOp

```

Comment:
💡  I personally prefer the rendering `Trustline`, but most mentions of trustlines in the .x files outside of comments is in the form `TrustLine` or `TRUST_LINE` or `trustLine`, and I think we should continue that consistency. There is the odd constant in the form `TRUSTLINE`. I defer to you and Core folks though.

Reply:
I updated the XDR to use `TrustLine` and `trustLine`, but I still need to address `TRUST_LINE` vs `TRUSTLINE` (`SET_TRUSTLINE_FLAGS_NO_TRUST_LINE` looks very weird).

Reply:
We have used _DOES_NOT_EXIST I think else. Maybe that would suffice?

Reply:
I think it should stay `_NO_TRUST_LINE` since we already use that when a trustline is missing in other operations. I changed `TRUSTLINE` to `TRUST_LINE` in Stellar-transaction.x.